src/dag/transitive.js

```diff
@@ -0,0 +1,104 @@
+const {NotFoundGraphError} = require('graphology');
```

Comment:
The standard library must not depend on `graphology` because there exists different implementations in the wild. Just throw an Error like here: https://github.com/graphology/graphology/blob/master/src/utils/infer-type.js#L18